### PR TITLE
REF: random functions in alphabet order

### PR DESCRIPTION
**Pull request by samir-nasibli**
## Description
* changed functions order

### CODE REVIEW

File: dpnp/random/_random.pyx [hunk 1]
```diff
@@ -300,114 +300,114 @@ cpdef dparray dpnp_gumbel(double loc, double scale, size):
...
-    univariate negative binomial distribution for a given parameter of the distribution
-    `a` and success probability `p` of a single trial.
+    Returns an array populated with samples from beta distribution.
+    `dpnp_laplace` generates a matrix filled with random floats sampled from a
+    univariate laplace distribution.
 
     """
 
-    dtype = numpy.int32
+    dtype = numpy.float64
```
Comment (shssf) [inline dpnp/random/_random.pyx:311]: @samir-nasibli Please use DPNP specific types as much as possible
https://github.com/IntelPython/dpnp/blob/4664aac456d8d20f6f115292ea1ccabf3af403d9/dpnp/dpnp_iface_types.py#L46

Comment (shssf) [reply dpnp/random/_random.pyx:311]: I see other same issues below

File: dpnp/random/dpnp_iface_random.py [hunk 2]
```diff
@@ -496,6 +437,65 @@ def gamma(shape, scale=1.0, size=None):
     return call_origin(numpy.random.gamma, shape, scale, size)
 
 
+def geometric(p, size=None):
```
Comment (shssf) [inline dpnp/random/dpnp_iface_random.py:440]: it is not good idea to mix "refactoring" with "development".
No changes needed. Please separate such things in future.

Comment (samir-nasibli) [reply dpnp/random/dpnp_iface_random.py:440]: There is no "development" here. I just changed places of some functions. Just Github is going crazy. There is only refactoring here. The refactoring idea is reflected in the PR title.

File: dpnp/random/_random.pyx (hunk 1 above)
Comment (samir-nasibli) [reply dpnp/random/_random.pyx:311]: If it is critical, I can change it on other PR.

**Pull request merged**